src/cpu/core_dynrec/risc_ppc64le.h

```diff
@@ -542,11 +542,11 @@ static void inline gen_call_function_raw(void *func, bool fastcall = true)
 // generate a call to a function with paramcount parameters
 // note: the parameters are loaded in the architecture specific way
 // using the gen_load_param_ functions below
-static uint64_t inline gen_call_function_setup(void *func,
+static inline const uint8_t* gen_call_function_setup(void *func,
                                              Bitu paramcount,
                                              bool fastcall = false)
 {
-	uint64_t proc_addr=(uint64_t)cache.pos;
+	const uint8_t* proc_addr=(const uint8_t*)cache.pos;
```

Comment:
Can avoid the C-cast because the `cache.pos` is already this type, and use `const auto` instead.

dyn_cache.h:

```c++
static struct {
    ...
	const uint8_t* pos = {}; // position in the cache block
} cache = {};
```

Reply:
Same comment for all below `(const uint8_t*)` casts of `cache.pos` (and cache.pos - 4).

Reply:
done